docker/server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
+
+RUN true \
```

Comment:
I like this construct, because it is friendly both to SCMs and to some text editors s.a. vim. If it disturbs you, I can of course remove it.